docs/docs/05-components/tooltips.md

```diff
@@ -17,7 +17,7 @@ Use Blazor Bootstrap tooltip component to add custom tooltips to your web pages.
 |--|--|--|--|--|--|
 | ChildContent | RenderFragment | Specifies the content to be rendered inside this. | | | 1.0.0 |
 | Color | `TooltipColor` | Gets or sets the tooltip color. | | `TooltipColor.None` | 1.10.0 |
-| Placement | Specifies the tooltip placement. Default is top right. | | `TooltipPlacement.Top` | 1.0.0 |
+| Placement | Specifies the tooltip placement. Default is top right. | | | `TooltipPlacement.Top` | 1.0.0 |
```

Comment:
@fbedogni Type column is missing. Please update as below.

| Placement | `TooltipPlacement` | Specifies the tooltip placement. Default is top right. | | `TooltipPlacement.Top` | 1.0.0 |